### PR TITLE
6 feature shop highlighting

### PR DESCRIPTION
**Pull request by HellInsider**
Now shop highlighting works. You can see it on the 2nd floor, while tapping on the screen (only in mobie mode).

### CODE REVIEW

File: code/device/viewer.h [hunk 1]
```diff
@@ -70,6 +74,10 @@ class viewer: public QGraphicsView
 
     std::map<QString, QGraphicsSvgItem*> unstableVisibleItems;
     std::map<QString, QGraphicsSvgItem*> selectableItems;
+
+    QGraphicsSvgItem* recoloredItem;    // Later I will make it to vector. IDD
+    QColor highlightColor = Qt::cyan;   // Color of highlighting. IDD
```
Comment (kirillova-ak5) [inline code/device/viewer.h:79]: static const?

File: code/core/viewer.cpp [hunk 2]
```diff
@@ -80,6 +81,27 @@ void viewer::AddSelectable(QString id)
     mapScene->addItem(selectableItems[id]);
 }
 
+void viewer::HighlightShop(QString id)
+{
+    qDebug() << selectableItems[id];
+    recoloredItem = selectableItems[id];
```
Comment (kirillova-ak5) [inline code/core/viewer.cpp:90]: If the map contains no item with key "id", the operator[] inserts a default-constructed value into the map with key "id", but i think it is not what we need and better will just do nothing

File: code/core/viewer.cpp [hunk 3]
```diff
@@ -80,6 +81,27 @@ void viewer::AddSelectable(QString id)
     mapScene->addItem(selectableItems[id]);
 }
 
+void viewer::HighlightShop(QString id)
+{
+    qDebug() << selectableItems[id];
```
Comment (kirillova-ak5) [inline code/core/viewer.cpp]: debug output can be deleted

File: code/device/viewer.h (hunk 1 above)
Comment (HellInsider) [reply code/device/viewer.h:79]: I can't set Qcolor as static or const because Qt swears me. 
Making recoloredItem to static will not have any effect.

**Pull request merged**